src/main/kotlin/frc/robot/subsystems/ReefCommands.kt

```diff
@@ -49,15 +51,21 @@ private fun visualizeCoralOuttake(): Command =
         )
     })
 
-private fun scoreCoral(endTrigger: Trigger): Command =
+private fun scoreCoral(
+    endTrigger: Trigger,
+    slowOuttake: Boolean = false
+): Command =
     sequence(
         waitUntil(endTrigger),
-        gripper
-            .outtake()
-            .withTimeout(CORAL_OUTTAKE_TIMEOUT)
-            .alongWith(
-                visualizeCoralOuttake().onlyIf { CURRENT_MODE != Mode.REAL }
-            ),
+        if (slowOuttake)
+            gripper.slowOuttake().withTimeout(CORAL_OUTTAKE_TIMEOUT)
+        else
+            gripper
+                .outtake()
+                .withTimeout(CORAL_OUTTAKE_TIMEOUT)
+                .alongWith(
+                    visualizeCoralOuttake().onlyIf { CURRENT_MODE != Mode.REAL }
```

Comment:
Are you sure the visualization should only be run if this is a regular outtake?
If not, you can combine the outtake timeout:
```suggestion
            sequence(
        waitUntil(endTrigger),
        (if (slowOuttake) gripper.slowOuttake() else gripper.outtake())
            .withTimeout(CORAL_OUTTAKE_TIMEOUT)
            .alongWith(
                visualizeCoralOuttake().onlyIf { CURRENT_MODE != Mode.REAL }
            ),
        moveDefaultPosition()
    )
```